pkg/endpointmanager/subscribe.go

```diff
@@ -10,12 +10,16 @@ type Subscriber interface {
 	// EndpointCreated is called at the end of endpoint creation.
 	// Implementations must not attempt write operations on the
 	// EndpointManager from this callback.
-	EndpointCreated(ep *endpoint.Endpoint)
+	// epList is the list of all endpoints that exist after the creation of
+	// the new endpoint.
+	EndpointCreated(ep *endpoint.Endpoint, epList []*endpoint.Endpoint)
```

Comment:
On the one hand a bit concerned with giving out pointers to all the EPs under a read lock, as it's then possible to change these in an unsafe way. On the other hand, that's been possible already, so it's not much worse.